source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-config.yaml

```diff
@@ -0,0 +1,95 @@
+stepnum: 1
+level: 4
+title: "Add ``transitionToAuth: true`` to the ``mongod`` configuration file."
+ref: modify-config-files
+pre: |
+  
+  Copy the existing {{program}} configuration file, giving it a distinct
+  name such as ``<filename>-secure``.
+  
+  You will use this new configuration file to transition the {{program}} to
+  enforce authentication in the sharded cluster. Retain the original
+  configuration file for backup purposes.
+
+  Add the following settings to the new configuration file.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 30 70
+
+     * - Setting
+       - Value
+
+     * - :setting:`security.transitionToAuth`
+       - ``true``
+
+     * - :setting:`security.keyFile`
+       - Path to the keyfile.
+
+         If using a different internal authentication mechanism, specify
+         settings appropriate for the mechanism.
+
+  The new configuration file should contain all of the configuration settings
+  previously used by the {{program}}, as well as the new security
+  settings.
+
+  .. code-block:: yaml
+
+     security:
+        transitionToAuth: true
+        keyFile: <path-to-keyfile>
+replacement:
+  program: :program:`mongod`
+---
+stepnum: 2
+level: 4
+ref: restart-mongod
+title: "Restart the ``mongod`` with the new configuration file."
+pre: |
+
+  If upgrading the a :term:`secondary`, connect to the :program:`mongod` and
```

Comment:
The organization is weird. 
Paragraph 1 (shutdown)
-or-
Paragraph 2 (shutdown)
-and must do step for either-
Paragraph 3 (the restart)

No reader is going to read let me read paragraph 1, and skip over a section and let me resume at paragraph 3 and repeat for each secondary. 
-------

Starting with the secondaries, restart each secondary, one at a time:

#. Connect to the mongod and use db.shutdownServer ...
#. Restart ....

After all the secondaries have been updated, restart the primary:

#.  COnnect ...
#.  Use rs.stepDown() to step down the primary and trigger ... After it steps down, use ... to ensure

#. shutdown 

#. Restart using ...